src/commands/command-fns/blog.js

```diff
@@ -0,0 +1,85 @@
+const got = require('got')
+const {getCommandArgs} = require('../utils')
+
+async function fetchArticles() {
+  const response = await got('https://kentcdodds.com/blog.json')
+
+  return JSON.parse(response.body)
+}
+
+function printArticles(articles) {
+  return articles
+    .map(article => {
+      return `- ${article.title}
+  <${article.productionUrl}>`
+    })
+    .join('\n')
+}
```

Comment:
Just some formatting:

```suggestion
function printArticles(articles) {
  return articles
    .map(article => `- ${article.title}\n  <${article.productionUrl}>`)
    .join('\n')
}
```